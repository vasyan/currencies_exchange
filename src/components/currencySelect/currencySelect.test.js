import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import CurrencySelect from './index'

describe('<CurrencySelect>', () => {
  it('should render', () => {
    const props = {
      children: <span>this is child</span>
    }

    expect(shallowToJson(shallow(<CurrencySelect />))).toMatchSnapshot()
    expect(
      shallowToJson(shallow(<CurrencySelect {...props} />))
    ).toMatchSnapshot()
  })
})
