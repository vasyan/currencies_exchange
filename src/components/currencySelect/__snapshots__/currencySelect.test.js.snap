// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrencySelect> should render 1`] = `
<div
  className="wrapper"
>
  <i
    className="icon_prev"
  />
  <div
    className="currency_name"
  >
    __CURRENCY_NAME__
  </div>
  <div
    className="children_wrapper"
  >
    <span>
      this is child
    </span>
  </div>
  <i
    className="icon_next"
  />
</div>
`;
